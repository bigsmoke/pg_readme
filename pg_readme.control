# This file is part of the `pg_readme` PostgreSQL extension.
# Copyright Â© 2022 Rowan Rodrik van der Molen.
#
# `pg_readme` is free software: you can redistribute it and/or modify it under
# the terms of the GNU Affero General Public License as published by the Free
# Software Foundation, either version 3 of the License, or (at your option) any
# later version.
#
# `pg_readme` is distributed in the hope that it will be useful, but WITHOUT
# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
# details.
#
# You should have received a copy of the GNU Affero General Public License
# along with `pg_readme`. If not, see <https://www.gnu.org/licenses/>.

default_version = '0.5.3'
requires = 'hstore'
superuser = false
relocatable = true
