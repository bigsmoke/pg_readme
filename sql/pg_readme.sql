create extension pg_readme cascade;

call test__pg_readme_pi_pseudo_attrs();
call test__pg_readme();
