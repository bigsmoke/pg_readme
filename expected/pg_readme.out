create extension pg_readme cascade;
NOTICE:  installing required extension "hstore"
call test__pg_readme_pi_pseudo_attrs();
call test__pg_readme();
